dipy/reconst/dti.py

```diff
@@ -756,8 +804,9 @@ def __init__(self, gtab, fit_method="WLS", return_S0_hat=False, *args,
             e_s = "The `min_signal` key-word argument needs to be strictly"
             e_s += " positive."
             raise ValueError(e_s)
+        self.extra = {}
 
-    def fit(self, data, mask=None):
+    def fit(self, data, mask=None, adjacency=0):
```

Comment:
I don't like the setting of adjacency to 0 here. I think it should be set to `False` per default and serve as a threshold/cutoff otherwise.

Reply:
Yeah that would be more intuitive for the user if it was `False`, thanks